.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+node_js:
+    - '0.10'
```

Comment:
@benpatterson I just updated to address your comments.  Travis builds now run with both 0.10 and 0.12.